pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -308,6 +301,12 @@ def init_ddp_connection(self, global_rank: Optional[int] = None, world_size: Opt
             log.info(f"initializing ddp: GLOBAL_RANK: {global_rank}, MEMBER: {global_rank + 1}/{world_size}")
             torch_distrib.init_process_group(self.torch_distributed_backend, rank=global_rank, world_size=world_size)
 
+            # on rank=0 let everyone know training is starting
+            rank_zero_info("-" * 100)
+            rank_zero_info(f"distributed_backend={self.distributed_backend}")
```

Comment:
Should this say accelerator since it's the argument the users set?

```suggestion
            rank_zero_info(f"accelerator={self.distributed_backend}")
```

Reply:
good point, or is it maybe supposed to say

`self.torch_distributed_backend`, e.g., either gloo or nccl?

now actually very unsure what is meant here ... hmm.

Reply:
made the choice to actually print the torch backend here. from the context, it makes much more sense. accelerator didn't seem right because we are dealing with plugins here.